static/js/modules/api.js

```diff
@@ -64,6 +90,35 @@ var filterLoadHandler = function(e) {
 };
 
 var loadSingleEntity = function(e) {
+    if (e.category === 'candidate') {
+        loadCandidate(e);
+    }
+    else {
+        loadCommittee(e);
+    }
+}
+
+var loadCandidate = function(e) {
+    var candidatePromise = callAPI(buildURL(e));
+
+    candidatePromise.done(function(data) {
+        var totalsPromises = [],
+            cycle = data.results[0].elections[0].election_year;
+
+        totalsPromises.push(callAPI('/rest/total/' + data.results[0].elections[0].primary_committee.committee_id + '?election_cycle=' + cycle));
+// data.results[0].elections[0].affiliated_committees[0].committee_id
```

Comment:
Oh good call. Thanks. That's for when I implement affiliated committee totals, but I should remove it from the code.